doc/bluealsa-aplay.1.rst

```diff
@@ -54,6 +54,9 @@ OPTIONS
     Select ALSA playback PCM device to use for audio output.
     The default is ``default``.
 
+    Available ALSA PCM devices can be listed by running **aplay -L**. You must
+    use the full PCM device name eg `front:CARD=Generic_1,DEV=0` not just `front`.
```

Comment:
OK, so I will use the power of repo owner and will settle this :D (unless someone will say "veto")

```suggestion
    On most Linux distributions, available ALSA PCM devices can be listed by running **aplay -L**.
```

Put it at the end of the `--pcm` help message (after "bluealsa-aplay does not perform..." paragraph; for most users this paragraph will be more important than the `aplay` hint).
Also, please add `aplay(1)` to the "SEE ALSO" list.

If you are worried that this simple statement might not be enough, add "EXAMPLES" section just before "SEE ALSO" (to be honest I'd be glad if you could provide some examples). In that section you can provide real life usage of `bluealsa-aplay` (2-3 examples) where you can use `-D` option with e.g. `front:CARD=USB,DEV=0`.